packages/editor-ui/src/mixins/externalHooks.ts

```diff
@@ -1,30 +1,167 @@
 import { IExternalHooks } from '@/Interface';
-import { useWebhooksStore } from '@/stores/webhooks';
-import { IDataObject } from 'n8n-workflow';
-import { Store } from 'pinia';
+import { IDataObject, INode, INodeTypeDescription } from 'n8n-workflow';
 import Vue from 'vue';
+import { Route } from 'vue-router/types/router';
+import {
+	AuthenticationModalEventData,
+	ExecutionFinishedEventData,
+	ExecutionStartedEventData,
+	ExpressionEditorEventsData,
+	InsertedItemFromExpEditorEventData,
+	NodeRemovedEventData,
+	NodeTypeChangedEventData,
+	OutputModeChangedEventData,
+	UpdatedWorkflowSettingsEventData,
+	UserSavedCredentialsEventData,
+} from '@/hooks/telemetry';
+
+export interface ExternalHooks {
```

Comment:
nice.. looks like you need to do a quick pass on the app and address issues
<img width="867" alt="Bildschirmfoto 2023-03-07 um 14 11 00" src="https://user-images.githubusercontent.com/4711238/223431897-abc9ce6a-903b-4602-8077-1c034aecba11.png">

<img width="851" alt="Bildschirmfoto 2023-03-07 um 14 12 09" src="https://user-images.githubusercontent.com/4711238/223432020-63954dc1-d5ef-44dc-af93-832c3de703b8.png">